test/fixtures.js

```diff
@@ -60,4 +60,109 @@ f.stack.binX = {
   }
 };
 
+// POINT
+
+f.points = {};
+
+f.points['1d_hor'] = {
+  "marktype": "point",
+  "encoding": {"x": {"name": "year","type": "O"}},
+  "config": {"singleHeight": 400,"singleWidth": 400,"largeBandMaxCardinality": 20},
```

Comment:
Reading this one more time, I am guessing that we actually do not need the custom config for any of these fixtures.  

(I saw that my old code has some of these leftover as well, so I probably I need clean to them up.)